django_atlassian/models/djira.py

```diff
@@ -41,6 +48,37 @@ def create_from_json(self, json):
         return self.model(**args)
 
 
+class JiraManagerMixin(JIRA):
+    def __init__(self, *args, **kwargs):
+        options = copy.copy(JIRA.DEFAULT_OPTIONS)
+        try:
+            db_alias = router.db_for_read(self.model)
+            db_settings = connections.databases[db_alias]
+            if db_settings['ENGINE'] == 'django_atlassian.backends.jira':
+                conn = connections[db_alias].connection
+                if conn:
+                    server = conn.uri
```

Comment:
This only works for a "static" connection, that is, connections defined on the database array. For connections "live" (like in an add-on) there's no password, only the jwt, therefore we can not use the basic auth. The other field is "security" IIRC.